src/Entity/ApplicationRequest/ApplicationRequest.php

```diff
@@ -30,12 +30,13 @@ abstract class ApplicationRequest implements ReferentTaggableEntity
      *
      * @ORM\Column
      *
-     * @Assert\NotBlank(message="application_request.first_name.not_blank")
+     * @Assert\NotBlank(message="application_request.first_name.not_blank", groups={"application_request"})
```

Comment:
I think you should use `Default` group too for keeping these constraints on Admin form validation (or just keep only one group for this entity - Default?)